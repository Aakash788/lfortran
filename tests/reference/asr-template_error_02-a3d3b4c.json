{
    "basename": "asr-template_error_02-a3d3b4c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/template_error_02.f90",
    "infile_hash": "b1a0a97aeb07a771bd88702bc021128598a7b6d7a1c32f4c3e53bf2b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-template_error_02-a3d3b4c.stderr",
    "stderr_hash": "36ad5a2a84a220e4f847ac59de37f5ffcbfb63110684aa64497b3f17",
    "returncode": 2
}