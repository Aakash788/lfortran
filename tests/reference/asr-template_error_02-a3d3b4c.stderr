syntax error: Token 'requires' (of type 'identifier') is unexpected here
  --> tests/errors/template_error_02.f90:15:9
   |
15 |         requires R(T, F)
   |         ^^^^^^^^ 
